examples/tx/weights_&_estimates.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * @title Weights & Estimates
```

Comment:
Btw never seen an ampersand in a source code filename. Are we sure this will not cause any errors down the line?

`examples/tx/weights_&_estimates.ts`

Reply:
Never mind, just realized it's an example file.